free-2.md

```diff
@@ -58,12 +123,41 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+My initial impression is that the "systems" aspect would pose a significant
+hurdle.  From there, you could start with a minimal language that only allows you
+to view and post limit orders and base much of the syntax off of another language
+like Python.  You could then build the language and create new syntax for the DSL.
+The complicated parts of the "systems" include extracting online data about the
+price of stocks, and allowing the user to specify a stock from an online list. 
+Also, there are many security issues involved, since you wouldn't want to be able
+to write a program that would trade somebody else's stocks.  To be a usable DSL,
+the security issue could be a very time-consuming task.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This idea is certainly bigger than the calendar DSL, and perhaps too big to be
+done in a semester.  Both the language and the systems part could balloon to be
+formidable tasks.  The systems part involves creating a secure way to buy and sell
+stocks, which could get complicated.  It would be possible to have the DSL merely
+calculate what the user _should_ do, and leave the actual trading for the user,
+but that would defeat the purpose of making it fast.  An ideal possibility would
+be to leverage some existing trading API that already deals with security.  Also,
+the language could get complicated, sense I would like to give the user as much
```

Comment:
This ties into my concern above; I think this might be a case where "dumbing it down" isn't necessarily the best decision.